installation/scripts/e2e-testing.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source ${ROOT_PATH}/utils.sh
+source ${ROOT_PATH}/testing-common.sh
+
+
+cleanupHelmE2ERelease () {
+    local release=$1
+    log 'Running cleanup'
+    helm del --purge  $release
+    while helm list --deleting 2>/dev/null | grep $release ; do
+        sleep 1
+        echo .
+    done
+}
+
+
+echo "-------------------------------"
+echo "- Ensure test Pods are deleted "
+echo "-------------------------------"
+
+cleanupHelmTestPods end-to-end
+cleanupE2EErr=$?
+
+
+
+echo "----------------------------"
+echo "- E2E Testing Kyma..."
+echo "----------------------------"
+
+
+for testcase in ../../tests/end-to-end/*/deploy/chart/*
```

Comment:
could you point me what will be tested here? I cannot find this files on the master branch

Reply:
Hi @aszecowka, the whole implementation of the e2e test is in this PR. 
https://github.com/kyma-project/kyma/pull/2619
However we are working on the modularization of the e2e test as it is explained in this issue, https://github.com/kyma-project/kyma/issues/2696